app/workers/ingest_cloudfront_log_worker.rb

```diff
@@ -0,0 +1,48 @@
+require "services"
```

Comment:
This seems extremely similar to the IIS log worker, and I think they could be DRYed up. All the specific stuff is in the `Transition::Import::Hits.from_cloudfront!` method, I think? But, as with my other comment, while it might be something to tidy up in future, it doesn't need to hold up the merge. Maybe open a ticket to DRY these up at some point in the future, maybe if we ever need to add another one.

Reply:
I have a few refactors I want to do on the IIS & Cloudfront parsers as we're extracting far more info from the log lines than we actually need.

Reply:
There is likely to be a couple more log formats (Default apache and nginx log formats. One of those would have helped UKRI bu they worked around that).